fast64_internal/oot/scene/properties.py

```diff
@@ -270,10 +270,9 @@ class OOTSceneHeaderProperty(PropertyGroup):
     # not been removed as doing so would break any existing scenes made with this
     # type of cutscene data.
     csEndFrame: IntProperty(name="End Frame", min=0, default=100)
-    csWriteTerminator: BoolProperty(name="Write Terminator (Code Execution)")
-    csTermIdx: IntProperty(name="Index", min=0)
-    csTermStart: IntProperty(name="Start Frm", min=0, default=99)
-    csTermEnd: IntProperty(name="End Frm", min=0, default=100)
+    csUseDestination: BoolProperty(name="Write Terminator (Code Execution)")
+    csDestination: IntProperty(name="Index", min=0)
+    csDestinationStartFrame: IntProperty(name="Start Frm", min=0, default=99)
```

Comment:
does frm need to be abbreviated?

Reply:
that's the deprecated code, I haven't updated it tbh I don't even know if everything still works for that

imo we should completely remove that legacy system and upgrade the things (though I don't have a cs in a scene to I can't really try if it'd work properly)

Reply:
should we just remove this thing? I'm not sure anyone still using that now idk if it's worth making the code to update properly